Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -9,13 +9,14 @@ var fileEncoding = require('./fileencoding.js');
 
 export async function substituteAppSettingsVariables(folderPath) {
     var configFiles = tl.glob(folderPath + "/**/*.config");
-    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections"];
+    var variableMap = varUtility.getVariableMap();
+    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections", "system.webServer"];
```

Comment:
AppSetttings and other tags were added in Substitution as per spec. System.WebServer contains attributes which need to be modified for Python web package.